configure.in

```diff
@@ -2487,6 +2487,18 @@ if test "$wxUSE_ZLIB" != "no" ; then
             wxUSE_ZLIB=sys
         fi
     fi
+
+    dnl We should test if built-in zlib library is avalable, preferably
+    dnl downloaded through --recurse-submodules option if cloned from git.
+    dnl Would be available when downloaded as a zipped or tarball release.
+    if test "$wxUSE_ZLIB" = "builtin" ; then
+        AC_CHECK_FILE(
+            $ac_confdir/src/zlib/zlib.h,,
+            [AC_MSG_ERROR([
+        === Configured to use built-in ZLIB library, but couldn't find required
```

Comment:
Minor, but why use `=== ` as the error message prefix? We don't seem to be using it anywhere else...

Also, even more minor, but I don't think `zlib` is ever capitalized.

Reply:
I think this warning prompt me to use "===":
```
config.status: executing wx-config commands
=== configuring in src/tiff (~/wxWidgets/build-release/src/tiff)
configure: WARNING: no configuration information is in src/tiff
```

The warning came from: 
`~/wxWidgets/build/autoconf_prepend-include/autoconf/status.m4`

But I'll change it if it doesn't look appropriate for an error message.